src/pages/components/OrdinoContactForm.tsx

```diff
@@ -1,47 +1,81 @@
+
+const useSubmitContactForm = () => {
+    const [inputs, setInputs] = React.useState({
+        name: '',
+        subject: 'I have a question',
+        message: '',
+        email: '',
+        mailTo: 'mailto:' + CONTACT_FORM_EMAIL,
+    });
+    const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        // persist event in order to access `event.target.name`
+        event.persist();
+        const {subject, message, name, email} = inputs;
+        let extra = subject || message ? '?' : '';
+        if (subject) {
+            extra += `subject=${encodeURIComponent(subject)}`;
+        }
+        if (message) {
+            // append name and email to the email body
+            // maybe it would be less confusing to completely remove these two fields and the user can add them directly to the message box
+            const body = `${message}${name ? '\n Name: ' + name : ''}${email ? '\n Email: ' + email : ''}`;
+            extra += `${subject ? '&' : ''}body=${encodeURIComponent(body)}`;
+        }
+        const newMailTo = 'mailto:' + CONTACT_FORM_EMAIL + extra;
+        setInputs((inputs) => ({...inputs, [event.target.name]: event.target.value, mailTo: newMailTo}));
+    };
+    return {
+        inputs,
+        handleInputChange,
+    };
+};
+
+export function OrdinoContactForm() {
+
+    const {inputs, handleInputChange} = useSubmitContactForm();
+    const {name, email, subject, message, mailTo} = inputs;
 
     return (
         <Card className="shadow-sm">
             <Card.Body>
                 <Card.Text>
-                    {'Do you have questions or found a bug, do not hasitate to contact us using the contact form below. You can also contact us by writing an email to '}
-                    <Card.Link href="mailto:ordino@caleydo.org.">ordino@caleydo.org</Card.Link> or posting a message in the Microsoft Team. We are glad to help you.
+                    {'Do you have questions or found a bug, do not hesitate to contact us using the contact form below. You can also contact us by writing an email to '}
+                    <Card.Link href="mailto:ordino@caleydo.org.">ordino@caleydo.org</Card.Link>. We are glad to help you.
                  </Card.Text>
-                <Form onSubmit={() => handleSubmit()}>
+                <Form>
                     <Form.Group className="row-cols-md-3">
                         <Form.Label>Type of contact</Form.Label>
-                        <Form.Control as="select">
-                            <option>I have a question</option>
-                            <option>I want to report a bug</option>
-                            <option>Placeholder</option>
+                        <Form.Control onChange={handleInputChange} value={subject} name="subject" as="select">
+                            <option value="I have a question">I have a question</option>
+                            <option value="I want to report a bug">I want to report a bug</option>
```

Comment:
Please add a general feedback as first option:

```suggestion
                            <option value="I have some general feedback">I have some general feedback</option>
                            <option value="I have a question">I have a question</option>
                            <option value="I want to report a bug">I want to report a bug</option>
```